kore/src/Kore/ModelChecker/Step.hs

```diff
@@ -215,30 +210,28 @@ transitionRule
     unificationProcedure = Unification.unificationProcedure
 
     transitionComputeWeakNextHelper
-        :: [RewriteRule VariableName]
-        -> Pattern VariableName
+        :: [RewriteRule RewritingVariableName]
+        -> Pattern RewritingVariableName
         -> Transition m CommonProofState
     transitionComputeWeakNextHelper _ config
         | Pattern.isBottom config = return Proven
     transitionComputeWeakNextHelper rules config = do
         results <-
             Step.applyRewriteRulesParallel
                 unificationProcedure
-                (mkRewritingRule <$> rules)
-                (mkRewritingPattern config)
```

Comment:
Where did these calls get moved?

Reply:
In [applyRewriteRulesParallel](https://github.com/kframework/kore/pull/2325/files#diff-50f642270881a6eafba1d357d4963900648cb26db0e7863b446e000b6fb2940cR339)